### PR TITLE
Feature/quick start task counter

### PR DESCRIPTION
**Pull request by khaykov**
Part of #7707 

This PR adds:

- Quick Start task counter
- All task completion indicator (orange dot)
- Functionality to hide the quick start after all task have been skipped OR all tasks have been completed and the list was checked one last time.
- Improved design of the Quick Start fragment.

To test:

1. Complete quick start tasks and make sure the counter updates.
2. When all the tasks are completed, make sure the orange dot is displayed near the task counter.
3. After checking Quick Start list when an orange dot is visible make sure it disappears from the MySite options list.
4. Skipping all tasks from Quick Start list should immediately hide Quick Start row from MySite fragment (@SylvesterWilmott could you confirm that this is intended behavior?)

PS: You can reset quick start flag with `AppPrefs.setQuickStartActive(true);`

Orange dot of doom
screenshot-2018-07-25_23 53 31 796

Updated Quick Start task likst design:
screenshot-2018-07-25_23 57 19 73

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java [hunk 1]
```diff
@@ -403,6 +410,21 @@ public void onClick(View v) {
...
+                    AppPrefs.setQuickStartActive(false);
+                    mQuickStartContainer.setVisibility(View.GONE);
+                }
+
+                ActivityLauncher.viewQuickStartForResult(getActivity());
+            }
+        });
+
+        mToolbar = rootView.findViewById(R.id.toolbar_main);
+        mToolbar.setTitle(mToolbarTitle);
```
Comment (theck13) [inline WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java]: These two `mToolbar` statements are [duplicated in `onCreateView`](https://github.com/wordpress-mobile/WordPress-Android/blob/20730178d7e9eb565de4118d75d55a1db345b264/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java#L236-L237).  I think they may have been accidentally copied/pasted and can be removed.

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java [hunk 2]
```diff
@@ -697,4 +700,13 @@ public static boolean getShouldTrackMagicLinkSignup() {
     public static void removeShouldTrackMagicLinkSignup() {
         remove(DeletablePrefKey.SHOULD_TRACK_MAGIC_LINK_SIGNUP);
     }
+
+    public static void setQuickStartActive(Boolean isActive) {
+        setBoolean(DeletablePrefKey.IS_QUICK_START_ACTIVE, isActive);
+    }
+
+    // TODO quick start is set to true by default for testing purposes. Remove in prod.
```
Comment (theck13) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java]: Is there a list of things to do before merging the feature branch into `develop`?  If not, we might want to start one so things like this and the other `// TODO check if Quick Start completed` items don't sneak into the release.

Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java]: I added the TODO list to master issue 👍

**Comment by khaykov** [comment]
Thank for catching all the things (especially the compound drawable crash!)! They snuck in with conflicts during rebase that came from splitting of the original QuickStart PR.

Everything should be fixed by now, and I also added the completion for View Site task when you tap on a site card.

**Comment by SylvesterWilmott** [comment]
@khaykov 

>Skipping all tasks from Quick Start list should immediately hide Quick Start row from MySite fragment (@SylvesterWilmott could you confirm that this is intended behavior?)

Totally missed this sorry. You got it right, it will immediately check everything and then hide the row when the user backs out of the view.

**Pull request merged**